docs/next.config.js

```diff
@@ -27,7 +28,14 @@ module.exports = withCSS({
     };
     config.module.rules.push({
       test: /.mdx?$/, // load both .md and .mdx files
-      use: [babelMdxLoader, '@mdx-js/loader', join(__dirname, './common/md-loader')],
+      use: [
+        babelMdxLoader,
+        {
+          loader: '@mdx-js/loader',
+          options: { mdPlugins: [headings] },
```

Comment:
Also the `mdPlugins` are deprecated in v1, I think you can use `remarkPlugins`. But I'm not sure if the plugin is using the rehype or remark AST.

Reply:
This is the thing I also mentioned on your PR - it was badly documented on their docs. When you merge your MDX update PR, I'll fix it ASAP 😉